data_mounts_config.json

```diff
@@ -1,18 +1,26 @@
 [
     {
         "purpose": "software",
-	"blk_device": "/dev/BOGUS",
+	"blk_device": "/dev/mapper/<WWID>",
 	"name": "u01",
+	"num": "1",
```

Comment:
I think these partition changes probably belong to PR #62 where they are discussed